lib/maze/client/selenium/bb_browsers.yml

```
@@ -65,16 +65,16 @@ edge_latest:
   version: 'latest'
   resolution: '1920x1080'
 
-safari_16:
-  platform: 'macOS'
+safari_18:
+  platforms: 'macOS'
```

I think this was an unintentional change?
```suggestion
  platform: 'macOS'
```